Libraries/LibWeb/Streams/CountQueuingStrategy.cpp

```diff
@@ -34,7 +34,9 @@ CountQueuingStrategy::~CountQueuingStrategy() = default;
 GC::Ref<WebIDL::CallbackType> CountQueuingStrategy::size()
 {
     // 1. Return this's relevant global object's count queuing strategy size function.
-    return verify_cast<HTML::Window>(HTML::relevant_global_object(*this)).count_queuing_strategy_size_function();
+    auto* global = dynamic_cast<HTML::UniversalGlobalScopeMixin*>(&HTML::relevant_global_object(*this));
```

Comment:
hm. sad that this is a dynamic cast. I suppose refactoring the class heirarchy so that all the globals get their PlatformObject-ness from the UGS is a bit out of scope though.

Reply:
Yeah, I was thinking about it at one stage but I left it as I thought it could get a bit messy with introduction of things like AudioWorklets or whatnot with all of the different mixins we need